pkg/web/client.go

```diff
@@ -65,12 +69,47 @@ func NewClient(dockerCommand command.Command, client *http.Client) *Client {
 	}
 }
 
-func (c *Client) PostNewVersion(ctx context.Context, image string, weights []File, files []File) error {
+func (c *Client) PostBuildStart(ctx context.Context, imageHash string, buildTime time.Duration) error {
+	jsonBody := map[string]any{
+		"image_hash":      imageHash,
+		"build_time":      types.Duration(buildTime).String(),
+		"push_start_time": time.Now().UTC(),
+	}
+
+	jsonData, err := json.Marshal(jsonBody)
+	if err != nil {
+		return util.WrapError(err, "failed to marshal JSON for build start")
+	}
+
+	url := webBaseURL()
+	url.Path = strings.Join([]string{"", "api", "models", "build-start"}, "/")
```

Comment:
nit: Given there's no variable data in this string, I think it should be a `const` instead.